internal/cmd/group_tokens.go

```diff
@@ -120,3 +130,18 @@ var groupCreateTokenCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func validateDBNames(dbNames []string) ([]string, error) {
```

Comment:
⛏️: you can simplify the function to:
```suggestion
func validateDBNames(dbNames []string) error {
```